packages/typespec-ts/src/index.ts

```diff
@@ -372,7 +376,12 @@ export async function $onEmit(context: EmitContext) {
     console.timeEnd("onEmit: generate files");
     console.timeEnd("onEmit: generate modular sources");
   }
-
+  function buildMetaDataJson() {
+    return {
+      path: "metadata.json",
+      content: `{"api-version" : "${apiVersion}","emitter-version": "0.38.6"}`
```

Comment:
pls get this from tcgc not build this by ourself.

![image](https://github.com/user-attachments/assets/59b764f9-ab9a-43d3-86f7-5b82aedd1dd3)